src/core/mdn-annotation.js

```diff
@@ -43,42 +41,44 @@ async function loadStyle() {
 
 function insertMDNBox(node) {
   const targetAncestor = node.closest("section");
```

Comment:
I wonder if we should check `targetAncestor` not being null too? It could be an authoring mistake.